### ISSUE TITLE
geo-query is not compatible with composer/installers newer than ~1.0.0

### ISSUE THREAD

**Issue by wujekbogdan** [birgire/geo-query#10]
I install WordPress with Composer (thank's to the awesome [Bedrock](https://roots.io/bedrock/) tool). Bedrock already uses composer/installers v ^1.4. geo-query depends on composer/installers too, it requires version ~1.0.0 that causes a conflict.

When trying to add  geo-query to my project Composer complains that:

```  
Problem 1
    - Installation request for birgir/geo-query ^0.0.6 -> satisfiable by birgir/geo-query[0.0.6].
    - birgir/geo-query 0.0.6 requires composer/installers ~1.0.0 -> satisfiable by composer/installers[1.0.x-dev, v1.0.0, v1.0.1, v1.0.10, v1.0.11, v1.0.12, v1.0.13, v1.0.14, v1.0.15, v1.0.16, v1.0.17, v1.0.18, v1.0.19, v1.0.2, v1.0.20, v1.0.21, v1.0.22, v1.0.23, v1.0.24, v1.0.25, v1.0.3, v1.0.4, v1.0.5, v1.0.6, v1.0.7, v1.0.8, v1.0.9] but these conflict with your requirements or minimum-stability.
```

Does it really mean that geo-query requires composer/installers that's not newer than ~1.0.0? Or is it just an oversight in the composer.json file? I suppose that it's safe to upgrade that dependency.

**Comment by birgire** [comment]
@wujekbogdan thank you for bringing that up.

We should definitely update the `composer.json` here.

I'm not using anything special from the autoloaders that need restriction to older versions. 

I'm thinking `^1.0.0` instead of `~1.0.0`, so the range would be `>= 1.0.0` and `< 2.0.0`, to stay within possible breaking changes of version `2`.

Thoughts?

**Comment by wujekbogdan** [comment]
@birgire 
Yes, IMO `~1.0.0` should be replaced with `^1.0.0`. I'm looking at [composer/installers CHANGELOG.md](https://github.com/composer/installers/blob/master/CHANGELOG.md) and I can't see anything there that could cause any problems. They're only adding stuff in new releases, there are no breaking changes.

**Comment by birgire** [comment]
```
Changelog
0.0.7 (2018-06-27)

Fixed #10. Use ^1.0.0 for composer installer. Props @wujekbogdan

```

Thanks
